### DISCUSSION TITLE
Restart schedule on deploy?

### DISCUSSION THREAD

**Discussion by wrabit** [spatie/laravel-short-schedule#57] [Q&A]
Should there be a need to restart the short schedule command on each deploy for any reason like package updates, updates to commands that it runs?

**Comment by indigoram89** [comment]
How to restart the process after deploy?

**Comment by wrabit** [comment]
I did figure out a way, add this to a post-deploy script:

`pgrep -f 'short-schedule:run' | xargs kill 2>/dev/null`

**Reply by indigoram89** [reply]
Thank you!

**Reply by wrabit** [reply]
This assumes you have it monitored with supervisor

**Reply by indigoram89** [reply]
Yes, of course. We use laravel forge for it =)

**Reply by wrabit** [reply]
We found using this package was driving our cpu sky high so we opted for supervising an artisan command, with the specific < 1 min interval logic within that. Probably good for light work though.

**Reply by indigoram89** [reply]
Yes, we also decided not to use the package. We want to try use a task scheduling to run queue jobs with delay =)

For example (command):

// run the job now
dispatch(new ExampleJob);

// and every 10 seconds this minute
for ($i = 1; $i <= 5; $i++ ) {
    dispatch(new ExampleJob)->delay($i * 10);
}

**Reply by wrabit** [reply]
Yes that will probably do, I've seen it before

**Reply by jasonchong3329** [reply]
> pgrep -f 'short-schedule:run' | xargs kill 2>/dev/null

hello , how to use laravel forgot it? cache:forget short-schedule:run ?
